src/main/connector/system/Linux/Linux.yaml

```diff
@@ -0,0 +1,156 @@
+extends:
+- ../System/System
+metrics:
+  system.service.loaded:
+    description: Service Loaded State
+    type:
+      stateSet:
+        - loaded
+        - not-found
+  system.service.active:
+    description: Service Active State
+    type:
+      stateSet:
+        - active
+        - inactive
+  system.service.sub:
+    description: Service Sub State
+    type:
+      stateSet:
+        - running
+        - dead
+        - exited
+connector:
+  displayName: "LinuxOS"
+  platform: "Any platform with LinuxOS"
+  reliesOn: "Linux OsCommands"
+  information: "Gives OS specific information and metrics"
+  detection:
+    connectionTypes:
+    - remote
+    - local
+    appliesTo:
+      - "linux"
+    criteria:
+      - type: osCommand
+        commandLine: "/usr/bin/uname -o"
+        expectedResult: "GNU/Linux"
+        errorMessage: "Not a valid Linux host."
+monitors:
+  cpu:
+    simple:
+      type: multiInstance
+      sources:
+        source(1):
+          # cpuId;user;nice;system;idle;iowait
+          type: osCommand
+          commandLine: /usr/bin/cat /proc/stat
+          computes:
+            - type: awk
+              script: /cpu[0-9]/ {sub("cpu",""); print $1 ";" $2 / 1000 ";" $3 / 1000 ";" $4 / 1000 ";" $5 / 1000 ";" $6 / 1000}

```

Comment:
It's okay to have inline scripts, but there's no need to force it on a single line.

```yaml
computes:
- type: awk
  script: |
    /cpu[0-9]/ {
      sub("cpu","")
      print $1 ";" $2 / 1000 ";" $3 / 1000 ";" $4 / 1000 ";" $5 / 1000 ";" $6 / 1000
    }
```